.gitignore

```diff
@@ -38,3 +38,6 @@ android.local.properties
 
 core/out/
 runners/**/out/
+
+**/node_modules
+**/build
```

Comment:
Belated nitpick: Are these excludes necessary?

`build/` directories are already excluded on line 30, and `node_modules/` is excluded specifically in [dokka-subprojects/plugin-base-frontend/.gitignore](https://github.com/Kotlin/dokka/blob/8e5c63d035ef44a269b8c43430f43f5c8eebfb63/dokka-subprojects/plugin-base-frontend/.gitignore)

Reply:
thanks, it's really unnecessary, removed it in https://github.com/Kotlin/dokka/pull/3714/files